src/Parallel/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ spectre_target_headers(
   Algorithm.hpp
   AlgorithmMetafunctions.hpp
   ArrayIndex.hpp
+  Callback.hpp
```

Comment:
commit message, there is no wrapper of CkCallback (you've restricted it to a particular test in the next commit)